tools/process_headers.zig

```diff
@@ -373,12 +373,10 @@ pub fn main() !void {
     };
     const generic_name = try std.fmt.allocPrint(allocator, "generic-{s}", .{abi_name});
 
-    // TODO compiler crashed when I wrote this the canonical way
-    var libc_targets: []const LibCTarget = undefined;
-    switch (vendor) {
-        .musl => libc_targets = &musl_targets,
-        .glibc => libc_targets = &glibc_targets,
-    }
+    const libc_targets: []const LibCTarget = switch (vendor) {
+        .musl => &musl_targets,
+        .glibc => &glibc_targets,
+    };
```

Comment:
```suggestion
    // TODO compiler crashed when I wrote this the canonical way
    var libc_targets: []const LibCTarget = undefined;
    switch (vendor) {
        .musl => libc_targets = &musl_targets,
        .glibc => libc_targets = &glibc_targets,
    }
```

I guess just accept this suggestion, which is effectively a revert, and then we'll just squash the PR when merging.